custom_components/kia_uvo/Vehicle.py

```diff
@@ -53,6 +53,7 @@ def __init__(
 
     async def update(self):
         try:
+            previous_vehicle_status = self.get_child_value("vehicleStatus")
```

Comment:
can we follow same naming below (current_vehicle_location). I am a bit obsessed with naming :) 
![image](https://user-images.githubusercontent.com/11085566/142633736-5bb509f4-d52d-413f-bbc7-4f02cd3cd4a8.png)

Reply:
@fuatakgun I moved the definition of the currentLocation below the impl update call, I think it isn't so much a naming consistency issue as a bug. I couldn't think of a reason to update the location using the previous location during an update instead of the current.

Reply:
There is no reason to call external endpoint if location coordinates have not changed, right?

Reply:
My comment is about naming; if we call current location as current, i was saying we should call current vehicle data.

Reply:
ic, it didn't make sense. until I saw the parameter name is old_vehicle_location, previous_vehicle_location seems more inline with old_vehicle_location then current, and make more sense without having to understand it's use.